package.json

```diff
@@ -157,7 +157,7 @@
     "rollup": "2.76.0",
     "ssri": "10.0.4",
     "stream-splitter": "0.3.2",
-    "strip-ansi": "7.1.0",
+    "strip-ansi": "6.0.1",
```

Comment:
Why did we need to downgrade here?

Reply:
it looks like strip-ansi is only used in `npm run test` the rest worked fine and the github action doesn't run the tests and I only tested apps and modules after updating the package.json the last time. 

I can remove the test for now so it will only be the one `webp` line and can add the test with the downgrade later